tests/local/delta/test_DeltaDatabaseSpec.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+
+from spetlr import Configurator
+from spetlr.configurator.sql.parse_sql import parse_sql_code_to_config
+from spetlr.deltaspec.DeltaDatabaseSpec import DeltaDatabaseSpec
+
+
+class TestTableSpecConversions(unittest.TestCase):
```

Comment:
do you test some of the places where you expect raises?

Reply:
i added some tests in the integration test for DetlaTableSpec